app/static/js/events/scheduler.js

```diff
@@ -259,18 +260,31 @@ function addSessionToTimeline(sessionRef, position, shouldBroadcast) {
     updateColor(sessionRefObject.$sessionElement, sessionRefObject.session.track);
 
     var $mobileSessionElement = $(mobileSessionTemplate);    
-    var content = sessionRefObject.session.title + " | ";
-    var speakers = [];
-    _.each(sessionRefObject.session.speakers, function(speaker) {
-        speakers.push(speaker.name);
-    });
-    content += speakers.join(', ');
     $mobileSessionElement.find('.time').text(sessionRefObject.session.start_time.format('hh:mm A'));
-    $mobileSessionElement.find('.event').text(content);
-    $mobileSessionElement.find('.event').attr("data-target", "#session-track-details"+sessionRefObject.session.id);
+    $mobileSessionElement.find('.title').text(sessionRefObject.session.title);
     $mobileSessionElement.find('.session-track-details').attr("id", "session-track-details"+sessionRefObject.session.id);
-    $mobileSessionElement.find('.session-speakers').text("Speakers: " + speakers.join(', '));
+    $mobileSessionElement.find('.event').attr("data-target", "#session-track-details"+sessionRefObject.session.id + ",#session-track-short-details"+sessionRefObject.session.id);
```

Comment:
Isn't there a way to avoid two data target? I think having two data targets reduces code readability. If there isn't then will go with this. If there is, I would prefer to see that solution